gtsfm/multi_view_optimizer.py

```diff
@@ -33,25 +34,25 @@ def __init__(
     def create_computation_graph(
         self,
         num_images: int,
-        keypoints_graph: List[Delayed],
-        i2Ri1_graph: Dict[Tuple[int, int], Delayed],
-        i2Ui1_graph: Dict[Tuple[int, int], Delayed],
-        v_corr_idxs_graph: Dict[Tuple[int, int], Delayed],
+        delayed_features: Dict[int, Tuple[Delayed, Delayed]],
```

Comment:
why do we use features here? 🤔

Reply:
Valid. Because tearing the frontend apart in caller is annoying. But we might revisit this in PR to master.